allennlp/modules/vision/image2image.py

```diff
@@ -0,0 +1,39 @@
+import torch
+from torch import nn, FloatTensor, IntTensor
+from typing import List
+
+from allennlp.common.registrable import Registrable
+
+
+class Image2ImageModule(nn.Module, Registrable):
```

Comment:
I don't remember - do we actually use this anywhere?  Is this an abstraction that we want to keep, or is it an old one that we're happy to get rid of now?